students/Hebdowska_Grazyna/lesson_06/the_number_of_distinct_words_in_some_test.py

```diff
@@ -0,0 +1,19 @@
+def distings_words(some_tesxt):
+    new_word = {}
+    for i in range(len(some_tesxt)):
+        new_text = some_tesxt[i].split(' ')
+        if '' in new_text:
+            new_text.remove('')
+        for n in new_text:
+            if n not in new_word.keys():
```

Comment:
Instead of checking for duplicates you might use sets() which includes only unique objects

Complete solution
```
all_strings = set()
for i in range(int(input('How many lines of text: '))):
    string = input('Line of text').split()
    all_strings.update(string)

print(len(all_strings))
```